CMakeLists.txt

```diff
@@ -16,7 +16,16 @@ set(VM_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/vm)
 
 if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     set(CLR_CMAKE_PLATFORM_UNIX 1)
-    set(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 1)
+    if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64)
+        set(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 1)
+    elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
```

Comment:
Unfortunately ARMv6 is going to be harder to support. So currently only the
RPi 2 will work at this time.

On Fri, 24 Jul 2015 09:27 Jostein Kjønigsen <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/dotnet/coreclr/pull/1210#discussion_r35404087>:
>
> > @@ -16,7 +16,16 @@ set(VM_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/vm)
> >
> >  if(CMAKE_SYSTEM_NAME STREQUAL Linux)
> >      set(CLR_CMAKE_PLATFORM_UNIX 1)
> > -    set(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 1)
> > +    if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64)
> > +        set(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 1)
> > +    elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
>
> Just to add some real world data. Raspbian (which is hard-float) running
> on a classic Raspberry pi (1) gives the following output:
>
> $ uname -a
> Linux Applepi 3.18.7+ #755 PREEMPT Thu Feb 12 17:14:31 GMT 2015 armv6l GNU/Linux
>
> Note there is no armv6hl in the output. Not sure if this is enough data
> to base a generalization on, but at least it's data :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/coreclr/pull/1210/files#r35404087>.
>